frontend/src/components/TimeTable/TimeTable.vue

```diff
@@ -51,7 +51,7 @@ function handleDownload(timetableTitle, timetableData) {
         </div>
         <div v-for="hour in maxHoursPerDayPerTimetable[timetableTitle]" :key="hour" class="row">
           <div class="col col--small col--vertical-center">{{ hour }}</div>
-          <div v-for="subject in timetableData.map(day => JSON.parse(day[hour - 1]))" :key="subject" class="col">
+          <div v-for="subject in timetableData.map(day => day[hour - 1])" :key="subject" class="col">
```

Comment:
Nitpick: the `in` part should be a computed so that the value is not recalculated on every DOM refresh tick.

Reply:
since this is nested `v-for`, creating computed would create more problems than it would solve, in my opinion. Even docs don't suggest creating computed for nested loops
https://vuejs.org/guide/essentials/list.html#displaying-filtered-sorted-results

Reply:
Sorry, did not notice it was nested.